site/stripe/DonateForm.tsx

```diff
@@ -366,26 +347,114 @@ export class DonateForm extends React.Component {
                     onloadCallback={this.onCaptchaLoad}
                     verifyCallback={this.onCaptchaVerify}
                 />
-
-                <button
-                    type="submit"
-                    className={classnames("owid-button", {
-                        disabled: this.isSubmitting,
-                    })}
-                    disabled={this.isLoading}
-                >
-                    Donate{" "}
-                    {this.amount ? `${this.currencySymbol}${this.amount}` : ""}{" "}
-                    {this.interval === "monthly" ? "per month" : ""}
-                </button>
-
-                <p className="note">
+                <div className="donation-payment">
+                    <button
```

Comment:
Just confirming [we're okay with the `owid-donate` Netlify function](https://github.com/owid/owid-grapher/issues/2751) for the time being? 

Should we make a ticket to migrate that, if Donation form v3 is a while away?

Reply:
yes that's still the plan. I created two issues to track this:
- #2908
- #2909